pom.xml

```diff
@@ -558,6 +563,8 @@ under the License.
                             <user.country>US</user.country>
                             <user.language>en</user.language>
                             <project.basedir>${project.basedir}</project.basedir>
+                            <!-- Expose as property so that test utils that spawn JVMs can pick it up. -->
+                            <surefire.module.config>${surefire.module.config}</surefire.module.config>
```

Comment:
please elaborate on why exposing the property

Reply:
That's actually a good question.
Initially I thought it might be useful in case there will be something similar to `org.apache.flink.runtime.testutils.TestJvmProcess` on connetors level. However after looking at connector's repos it seems not the case at least for the moment... So I will remove it